ring/spread_minimizing_token_generator.go

```diff
@@ -42,32 +45,36 @@ var (
 )
 
 type SpreadMinimizingConfig struct {
```

Comment:
Do we need this config at all? I would just pass everything to `NewSpreadMinimizingTokenGenerator()` for now. In practice it doesn't change the number of parameters to pass to `NewSpreadMinimizingTokenGenerator()` given this config struct has only 1 field.

Reply:
You are right, we don't need this config.
I have removed it, and used a slice of zones instead.